Configurations/90-team.conf

```diff
@@ -16,7 +16,8 @@
     "debug-erbridge" => {
         inherit_from     => [ "x86_64_asm" ],
         cc               => "gcc",
-        cflags           => combine("$gcc_devteam_warn -DBN_DEBUG -DCONF_DEBUG -m64 -DL_ENDIAN -DTERMIO -g",
+        cflags           => combine($devteam_common_warn,
```

Comment:
This needs adjustment/harmonization with new variable name. Alternatively just remove this target, it's no different from --debug --strict-warnings. Otherwise adjustment/harmonization applies to other cases here...